examples/ale_classification.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<ul><li>"which reflcts the non-linear nature of applying the softmax" -&gt; " due the softmax transformation applied to the logits" would be a bit more concise</li><li><em>in this case</em> should be in between commas</li><li>I think i.e. also takes a comma since if you were to be less fancy and say _that is_ you'd need a comma</li><li>Easier to read if we put a full stop of ; after <em>decrease in probability </em><strong><em>of </em></strong><em>(not to?) class </em>and a comma after <em>In fact </em>(or leave it out altogether).</li><li>Maybe worth stating that ALE values for the same interval or something (or the same feature vals) ...</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<ul><li>Seems like British English doesn't use comma after i.e.</li></ul>